torch_object_detection_segmenter.py

```diff
@@ -192,7 +192,7 @@ def _crop_image(img, target_size: Union[Tuple[int, int], int], top: int = None,
                     _img, top, left = _crop_image(img, target_size=target_size, top=top, left=left, how='precise')
                     _img = _move_channel_axis(np.asarray(_img).astype('float32'), -1, self.channel_axis)
                     label_name = self.label_name_map[label]
-                    logging.debug(
+                    JinaLogger.debug(
```

Comment:
see in the chore how u instantiate a JinaLogger...

Reply:
Adopted